Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -1422,6 +1422,678 @@ bool RenderStyle::diffRequiresLayerRepaint(const RenderStyle& style, bool isComp
     return false;
 }
 
+void RenderStyle::collectAffectedProperties(const RenderStyle& other, CSSPropertiesBitSet& changingProperties) const
+{
+    // This function conservatively answsers two things,
+    //   1. Whether we can use CSSTransitionAll fast path.
+    //   2. What CSS properties we should visit in the fast path.
+    // (2) is conservative. We do not need to precisely check equivalence before saying "this property needs to be visisted".
+    // We can extend this function to increase coverage of this fast path by adding more properties.
```

Comment:
As far as I can tell any missing property here will just fail to animate?

Reply:
Yes, for `transition all` case.